lib/identity_job_log_subscriber.rb

```diff
@@ -4,122 +4,142 @@
 class IdentityJobLogSubscriber < ActiveSupport::LogSubscriber
   def enqueue(event)
     job = event.payload[:job]
-    ex = event.payload[:exception_object]
 
-    json = default_attributes(event, job)
+    if !job.class.try(:skip_log_on_failure?)
```

Comment:
I tried viewing the PR with [`?w=1`](https://github.com/18F/identity-idp/pull/5848/files?w=1) to see the changes, and it was much more manageable, but another thing to consider would be an early return?

ex:

```suggestion
    return if job.class.try(:skip_log_on_failure?)
```